src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,9 @@
+package seedu.addressbook.data.person;
+
+/**
+ * A Printable interface for each type of contact information in the address book
+ */
+public interface Printable {
+
+    String getPrintableString();
```

Comment:
Based on your implementation of `getPrintableString()`, the method does not seem trivial.
In this case a header comment in required for clarity on the method's functionality.